### ISSUE TITLE
Use 'aria-expanded' attribute to label buttons that show and hide panels

### ISSUE THREAD

**Issue by klemay** [hypothesis/product-backlog#1110]
This issue was raised by our follow-up accessibility audit. 

The auditor has suggested that we implement this for the following buttons:

- The Show/Hide sidebar toggle
- The Help toggle at the top of the toolbar
- The More/Less button that shows on truncated annotations:

<img width="511" alt="Screen Shot 2020-05-20 at 1 42 22 PM" src="https://user-images.githubusercontent.com/12087705/82479072-e2934000-9a9f-11ea-808e-7c2f4b71c19a.png">

[We discussed this in Slack](https://hypothes-is.slack.com/archives/C8TPC8XMK/p1589993197063600?thread_ts=1589988329.060800&cid=C8TPC8XMK) and it seems reasonable.

**Comment by lyzadanger** [comment]
It confuses me as to why the "Help" toggle should have an `aria-expanded` attribute, but not the "Share annotations" toggle, which has identical functionality (show/hide a panel in the sidebar). Help?

**Comment by klemay** [comment]
Sounds like we need `aria-expanded` attribute on both - I will confirm with the auditor and update the issue text if needed!
